### PR TITLE
Few changes for corefx to follow arcade artifacts convention for packages

### PR DESCRIPTION
**Pull request by safern**

### CODE REVIEW

**Comment by safern** [comment]
Added NO MERGE label as we need to react in corefx and that change is not ready yet.

File: src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/Packaging.common.targets [hunk 1]
```diff
@@ -9,8 +9,8 @@
     <PackagingTaskDir Condition="'$(PackagingTaskDir)' == '' AND '$(MSBuildRuntimeType)' != 'core'">$(MSBuildThisFileDirectory)../tools/net472/</PackagingTaskDir>
     <RuntimeIdGraphDefinitionFile Condition="'$(RuntimeIdGraphDefinitionFile)' == ''">$(MSBuildThisFileDirectory)runtime.json</RuntimeIdGraphDefinitionFile>
 
-    <PackageOutputPath Condition="'$(PackageOutputPath)' == ''">$(BaseOutputPath)pkg/</PackageOutputPath>
-    <PackageReportDir Condition="'$(PackageReportDir)' == ''">$(PackageOutputPath)reports/</PackageReportDir>
+    <PackageOutputPath Condition="'$(PackageOutputPath)' == ''">$([MSBuild]::NormalizePath('$(BaseOutputPath)', 'pkg/'))</PackageOutputPath>
```
Comment (ericstj) [inline src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/Packaging.common.targets]: Use NormalizeDirectory and you don't need to put in the trailing slash.

**Comment by safern** [comment]
> Can you explain why you are using a different property now? ArtifactsPackagesDir

Since we're moving towards following arcade's convention, `PackageOutputPath` will point to `ArtifactsPackagesDir\{Shipping|NonShipping}` depending on the pkproj's settings, if it has `IsShipping=true` or not. However, we want the specs and reports to be all together at the root dir for packages rather than having 1 specs/reports directory on each `Shipping` and `NonShipping` directory, `ArtifactsPackagesDir` is that root dir.

**Comment by ViktorHofer** [comment]
OK 👌

**Comment by safern** [comment]
Merging as I will be sending a PR to corefx shortly.

**Pull request merged**